lib/home/schedule/schedule.dart

```diff
@@ -1,4 +1,7 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
```

Comment:
```suggestion
```

For some reason, I've noticed that the "widgets" library gets auto-imported a lot, even though everything in there is covered by "material".